R/estimate_grouplevel.R

```diff
@@ -49,7 +57,7 @@ estimate_grouplevel <- function(model, type = "random", ...) {
   # Extract params
   params <- parameters::model_parameters(
     model,
-    effects = ifelse(type == "random", "all", "total"),
+    effects = ifelse(type == "random", "random", "total"),
```

Comment:
We had `"all"` in our initial code (better said: you wrote `"all"` as option), and indeed we need it, I guess, to make it work with Bayesian models. Not sure why `"random"` doesn't work?

Reply:
We needed all before to grab the fixed effects and do the sum, but since this has been outsourced we might as well only pull random effects

Reply:
But this still doesn't work, see failing tests.

Reply:
I think we need to remove:

```r
  # TODO: improve / add new printing that groups by group/level?
  random <- as.data.frame(params[params$Effects == type, ])
```

We have no `Effects` column for `effects = "random")`.